src/test/ui/in-band-lifetimes/mismatched.nll.stderr

```diff
@@ -14,9 +14,9 @@ error[E0621]: explicit lifetime required in the type of `y`
   --> $DIR/mismatched.rs:14:42
    |
 LL | fn foo(x: &'a u32, y: &u32) -> &'a u32 { y } //~ ERROR explicit lifetime required
-   |                    -                     ^ lifetime `'a` required
-   |                    |
-   |                    consider changing the type of `y` to `&'a u32`
+   |                       ----               ^ lifetime `'a` required
+   |                       |
+   |                       help: consider changing the type of `y` to : `&'a u32`
```

Comment:
@estebank should we change this message? because with `:` it looks bit weird. Any suggestion?

Reply:
Lets reword it slightly, so it reads `help: add explicit lifetime `'a` to 'y''s type: '&'a u32'`, which is slightly better. The colon is always there for suggestions. If the message is long, or there are multiple possible suggestions, or the suggestion spans multiple lines the emitter will look like this:

```
error[E0621]: explicit lifetime required in the type of `y`
  --> $DIR/mismatched.rs:14:42
   |
LL | fn foo(x: &'a u32, y: &u32) -> &'a u32 { y } //~ ERROR explicit lifetime required
   |                                          ^ lifetime `'a` required
help: add explicit lifetime `'a` to `y`'s type
   |
XX | fn foo(x: &'a u32, y: &'a u32) -> &'a u32 { y }
   |                       ^^^^^^^
```

Because of this, wording needs to be able to work for both types of presentation.